QModManager/Utility/Dialog.cs

```diff
@@ -122,16 +156,19 @@ private static IEnumerator ShowDialogEnumerator(string error, Button leftButton,
 
                 confirmation.gameObject.GetComponentInChildren<Image>().sprite = sprite;
 
-                texts.Do(t => t.fontSize = t.fontSize + 2);
+                texts.Do(t => t.fontSize = t.fontSize + 4);
 
-                UnityEngine.Object.Destroy(couroutineHandler);
+                UnityEngine.Object.Destroy(coroutineHandler);
 
                 // Re-open the dialog if it is not closeable
-                if (cannotClose)
-                    Show(error, leftButton, rightButton, blue, cannotClose, .05f);
+                if ((leftButtonClicked && leftButton != Button.Close) || (!leftButtonClicked && rightButton != Button.Close)) Show();
```

Comment:
Please keep the code to execute on a separate line from the if-statement

Reply:
It's 7 characters